tests/Functional/Schema/SQLServerSchemaManagerTest.php

```diff
@@ -404,4 +404,31 @@ public function testPkOrdering(): void
         self::assertEquals('colB', $columns[0]);
         self::assertEquals('colA', $columns[1]);
     }
+
+    public function testListSchemaNames(): void
+    {
+        //Schemas db_* with schema_id>=16384 are backward compatible schemas:
+        //https://learn.microsoft.com/en-us/sql/relational-databases/security/authentication-access/ownership-and-user-schema-separation?view=sql-server-ver16#built-in-schemas-for-backward-compatibility
+        //and should't be listed because creating DDL object is not possible on such schemas
+        //Te same page contains information about schemas: dbo, guest, sys, INFORMATION_SCHEMA
+        //they are system fixed and can't be dropped or re-created so listing them for SchemaDiff doesn't make sense
```

Comment:
It's enough to add a short comment and the link to the method implementation, not the test. This comment looks too large to me. The remaining details could be moved to the pull request description.